config/manifests/bases/sriov-network-operator.clusterserviceversion.yaml

```diff
@@ -118,7 +118,7 @@ spec:
                 - name: NETWORK_RESOURCES_INJECTOR_IMAGE
                   value: quay.io/openshift/origin-sriov-dp-admission-controller:4.14
                 - name: OPERATOR_NAME
-                  value: sriov-network-operator
+                  value: quay.io/openshift/origin-sriov-network-operator:4.14
```

Comment:
this is the operator-name not the image ?

Reply:
thanks. fixed